.github/workflows/web_generator.yaml

```diff
@@ -0,0 +1,67 @@
+name: package:web_generator
+permissions: read-all
+
+on:
+  # Run CI on pushes to the main branch and on PRs.
+  push:
+    branches: [ main ]
+    paths:
+      - '.github/workflows/web_generator.yaml'
+      - 'web_generator/**'
+  pull_request:
+    paths:
+      - '.github/workflows/web_generator.yaml'
+      - 'web_generator/**'
+  schedule:
+    - cron: "0 0 * * 0"
+
+defaults:
+  run:
+    working-directory: web_generator/
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        sdk: [stable]
```

Comment:
We were testing `dev` before so maybe replace this with that? We could include `stable` if we want, but `dev` is more important since that's what users (us!) will likely use for this package.